docs/content/concepts/metadata-tags.mdx

```diff
@@ -40,6 +41,14 @@ How metadata is defined depends on whether you're using assets or ops and jobs:
 
 ---
 
+## How kinds work
+
+Kinds label and categorize definitions in Dagster. Notably, kinds correspond to prominently displayed icons in our visual tools (see XXX for supported visual kinds). An effective proxy for whether something should be a kind is whether or not it is an adjective in your day-to-day language and meaningfully identifies that definition. E.g. If "Is this a dbt asset or a databricks asset?" is a question in your team that would indicate that "dbt" and "spark" are good kinds for your team.
+
+In its implementation, kinds are system-defined tags, prefixed with the "dagster/kind/" prefix. As is true with all of our system tags, our UIs reserve the right to treat these tags specially, hiding them or promoting them as appropriate.
```

Comment:
> In its implementation, kinds are system-defined tags, prefixed with the "dagster/kind/" prefix.

👍  this is great! I liked the explicitness here and it's also good that we can gradually disclose the relation between kinds and tags.

Reply:
> However, I think this particular doc page would need more polish or IA rewrite to become "jobs to be done" and more complete -- we need to map the use cases to different concepts and their usage; for example, "categorize assets visually" -> "kinds", "annotate assets with arbitrary info for better filtering and searching" -> "tags", etc.

My explicit goal was to minimally add kinds to this page rather than to completely revamp.